orienteer-core/src/main/java/org/orienteer/core/service/Reload.java

```diff
@@ -0,0 +1,47 @@
+package org.orienteer.core.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.ServletException;
+
+/**
+ * @author Vitaliy Gonchar
+ */
+public class Reload implements Runnable {
```

Comment:
As commented below: you can extends this class from Thread to be able to call it as new Reload(..).start()